cranelift/codegen/src/isa/call_conv.rs

```diff
@@ -75,6 +80,27 @@ impl CallConv {
             _ => false,
         }
     }
+
+    /// Is the calling convention extending the Wasmtime ABI?
+    pub fn extends_wasmtime(self) -> bool {
+        match self {
+            Self::WasmtimeSystemV | Self::WasmtimeFastcall => true,
+            _ => false,
+        }
+    }
+
+    /// Returns the kind of unwind info used for this calling convention
+    pub fn unwind_info_kind(self) -> UnwindInfoKind {
+        match self {
+            #[cfg(feature = "unwind")]
+            CallConv::Fast | CallConv::Cold | CallConv::SystemV | CallConv::WasmtimeSystemV => {
+                UnwindInfoKind::SystemV
+            }
+            #[cfg(feature = "unwind")]
+            CallConv::WindowsFastcall | CallConv::WasmtimeFastcall => UnwindInfoKind::Windows,
+            _ => UnwindInfoKind::None,
```

Comment:
Can you please avoid this wildcard arm? This makes it harder to find all places that need updates when adding a new call conv.